src/plugins/compact-view/compact.less

```diff
@@ -211,3 +211,11 @@
     }
   }
 }
+
+.compact-view-enabled.hide-folder-toggles {
+  .dcg-action-toggle-secret-folder,
+  .dcg-action-toggle-in-front-of-everything {
+    display: none;
+    visibility: hidden;
```

Comment:
Is `visibility: hidden` needed? Should be redundant with `display: none` afaict.

Reply:
I wasn't sure if it was needed. I can remove it.

Reply:
It was redundant. I just removed it